src/Content/AttachmentLocal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Phlib\Mail\Content;
+
+use Phlib\Mail\Exception\RuntimeException;
+
+/**
+ * Attachment class used to add local files to a Mail object
+ *
+ * @package Phlib\Mail
+ */
+class AttachmentLocal extends Attachment
```

Comment:
Would this be better called FileAttachment or AttachmentFile? AttachmentLocal is confusing how it relates to an email attachment.

What's the difference between Attachment and AttachmentLocal. Is one Inline? If so, would naming it more clearly be better?